pyscf/pbc/df/df_jk.py

```diff
@@ -296,13 +296,15 @@ def get_k_kpts(mydf, dm_kpts, hermi=1, kpts=numpy.zeros((1,3)), kpts_band=None,
     skmoR = skmo2R = None
     if not mydf.force_dm_kbuild:
         if mo_coeff is not None:
-            if isinstance(mo_coeff[0], (list, tuple)):
+            if isinstance(mo_coeff[0], (list, tuple)) or (isinstance(mo_coeff[0], numpy.ndarray)
+                                                          and mo_coeff[0].ndim == 3):
```

Comment:
Is this check for unrestricted calculations or does this work for restricted ones, too, as ndim of  mo_coeff[0] changes then maybe?

Reply:
I think this is only related to the unrestricted calculation. i.e. for rhf / rks, it will not have ndim == 3

Reply:
Makes sense, thanks!